js-core-1/week-1/lesson.md

```diff
@@ -200,22 +200,20 @@ var difference = 10 - 2; // 8
 ```sh
 Number of students: 15
 Number of mentors: 8
-Total numnber of students and mentors: 23
+Total number of students and mentors: 23
 ```
 
-## Floats
-
-Numbers can be integers (whole numbers) or floats (numbers with a decimal).
+Numbers can be written with, or without decimals:
```

Comment:
Although `float` is not a type in JavaScript, I still prefer the previous wording as an introduction to this terminology.

Reply:
Will probably updated in a later commit (I agree with you here btw @ismaelgt )